static/styles/portico/portico.scss

```diff
@@ -392,7 +392,7 @@ img.screenshot {
 }
 
 #registration #pw_strength {
-    width: 325px;
+    width: calc(100% - 5px);
```

Comment:
I consider `calc()` to be a bad practice. The same can be achieved using either padding or margin.

Reply:
Adding margin/padding doesn't seem to work with `width: 100%;` as margins are not included in width so it cannot compress the element to be lesser than specified width and only shifts the element from one side. Adding padding is of no use as it only affects the area inside border.

Only way I can think of to replace `calc()`  is to mention width in pixels like before but then it would be needed to be changed again if form width is changed. Any other way to achieve this?

Reply:
With `box-sizing: border-box;` the padding is included in the width.

Reply:
Adding padding creates a space between strength bar and end-points.

Without padding:
![Screenshot_2020-08-11 Chat for distributed teams(1)](https://user-images.githubusercontent.com/39924567/89818955-45104f80-db68-11ea-85a5-14abcc6bceee.png)

With `padding: 0 5px;`:
![Screenshot_2020-08-11 Chat for distributed teams](https://user-images.githubusercontent.com/39924567/89818845-12665700-db68-11ea-8e5e-3ef1acd94397.png)

Reply:
Should I resolve this as I can't find any other way to do this?

Reply:
Yeah wrapping the element is not really worth it. We could however use `width: 98%;` which looks nearly identical.

Reply:
Yeah, seems good. Thanks.